app/actions/v3/service_instance_create_managed.rb

```diff
@@ -38,7 +38,11 @@ def precursor(message:)
           state: ManagedServiceInstance::IN_PROGRESS_STRING
         }
 
-        ManagedServiceInstance.new.tap do |i|
```

Comment:
This comment is for the beginning of the method...

I had a look where this `precursor` method is invoked. `create_managed` in `ServiceInstancesV3Controller` seems to be the only(?) place. As there the service plan is already fetched from the database, I'm wondering if it makes sense to add `service_plan` as a second (optional) parameter. If provided, fetching it from the database in this method is not necessary anymore, i.e. one query less.

Reply:
Added service_plan as a non-optional parameter. I also just found this one caller.